mocks/pdf.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const router = require('express').Router();
+
+router.post('/', (req, res, next) => {
+  if (req.body.template) {
+    res.setHeader('Content-Type', 'octet-stream');
+    res.status(201).send(Buffer(100));
```

Comment:
It might be easier just to have a dummy pdf file that gets sent back with `res.sendFile`